tests/contrib/awards/test_awards_resource.py

```diff
@@ -70,6 +70,13 @@ def test_awards_search(client, example_award, h, prefix):
     assert res.status_code == 200
     assert res.json["hits"]["total"] == 1
     assert res.json["sortBy"] == "newest"
+    assert res.json["aggregations"]["funders"]
+
+    funders_agg = res.json["aggregations"]["funders"]["buckets"][0]
+    assert funders_agg["key"] == "01ggx4157"
+    assert funders_agg["doc_count"] == 1
+    assert funders_agg["label"] == \
+        "European Organization for Nuclear Research (CH)"
```

Comment:
Can we add a test using both the facet and suggest? This will be closer to the "real-life" usage also.

Reply:
Done, split in two:
- test_suggest_sort (previous test).
- test_faceted_suggest, with a query that should return two but only returns one because it is filtered by funder.